qiskit/synthesis/linear/linear_depth_lnn.py

```diff
@@ -243,17 +243,17 @@ def synth_cnot_depth_line_kms(mat):
     Kutin, Moulton, Smithline method.
 
     Synthesis algorithm for linear reversible circuits from [1], Chapter 7.
```

Comment:
I think it should be "Section 7" and not "Chapter 7" since it's a paper, and not a book.